.travis.yml

```diff
@@ -0,0 +1,67 @@
+sudo: false
+dist: xenial
+language: node_js
+node_js: 8.11
+
+cache:
+  directories:
+    - node_modules
+
+addons:
+  firefox: "66.0"
+  chrome: stable
+
+jobs:
+  include:
+    - if: type = push
+      env: TEST_SUITE=unit_tests POLYMER=2
+    - if: type = push
+      env: TEST_SUITE=visual_tests POLYMER=2
+    - if: type = push
+      env: TEST_SUITE=unit_tests POLYMER=3
+    - if: type = pull_request
+      env: POLYMER=2
+      addons:
+        firefox: "66.0"
```

Comment:
You can use `latest`, the problem that forced us to pin this version has been fixed.

Reply:
Updated.